send/interceptor_test.go

```diff
@@ -0,0 +1,29 @@
+package send
+
+import (
+	"testing"
+
+	"github.com/cdr/grip/level"
```

Comment:
ah I guess we're using a different goimports for this project :thinking:

Reply:
yeah, I'm fine with that.